<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configui</class>
 <widget class="QDialog" name="configui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settings_pages">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="input_filter">
         <property name="text">
          <string>Filter Microphone with RNNoise</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="input_vad">
         <property name="text">
          <string>Use RNNoise for VAD</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="vad_cutoff_layout" stretch="1,1,2">
         <item>
          <widget class="QLabel" name="vad_cutoff_label">
           <property name="text">
            <string>VAD Cutoff</string>
           </property>
           <property name="buddy">
            <cstring>vad_cutoff</cstring>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="vad_cutoff_percentage">
           <property name="text">
            <string>70 %</string>
           </property>
           <property name="buddy">
            <cstring>vad_cutoff</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="vad_cutoff">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>70</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="vad_rolloff_layout" stretch="1,1,2">
         <item>
          <widget class="QLabel" name="vad_rolloff_label">
           <property name="text">
            <string>VAD Rolloff</string>
           </property>
           <property name="buddy">
            <cstring>vad_rolloff</cstring>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="vad_rolloff_time">
           <property name="text">
            <string>0 ms</string>
           </property>
           <property name="buddy">
            <cstring>vad_rolloff</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="vad_rolloff">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="input_agc">
         <property name="text">
          <string>Dynamic Automatic Gain Control</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="output">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="output_filter">
         <property name="text">
          <string>Filter entire Output using RNNoise</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Individual Filters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="uuids">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>NotoMono Nerd Font Mono</family>
          </font>
         </property>
         <property name="placeholderText">
          <string>UUIDs for individual filtering (Comma or Newline separated)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="footer" stretch="1,0,0,0">
     <item>
      <widget class="QLabel" name="credits">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;JUL14Ns Audio Mods by &lt;a href=&quot;https://julianimhof.de&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;JUL14N&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>settings_pages</tabstop>
  <tabstop>input_filter</tabstop>
  <tabstop>input_vad</tabstop>
  <tabstop>vad_cutoff</tabstop>
  <tabstop>vad_rolloff</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbApply</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>input_vad</sender>
   <signal>toggled(bool)</signal>
   <receiver>vad_cutoff_percentage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_vad</sender>
   <signal>toggled(bool)</signal>
   <receiver>vad_cutoff</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_vad</sender>
   <signal>toggled(bool)</signal>
   <receiver>vad_rolloff_time</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input_vad</sender>
   <signal>toggled(bool)</signal>
   <receiver>vad_rolloff</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_filter</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_filter</sender>
   <signal>toggled(bool)</signal>
   <receiver>uuids</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
